src/api/fetchData.js

```diff
@@ -1,3 +1,19 @@
+import { API_BASE_URL } from '../utils/constants';
```

Comment:
As we talked you're not using this function or file at all, please remove it.

Reply:
Solved on the last [commit](https://github.com/laulujan/react-apprenticeship-capstone4/pull/3/commits/ac1b3a08724a7062fb585b159efbda8325777b88).